source/val/validation_state.h

```diff
@@ -707,6 +707,10 @@ class ValidationState_t {
   // Validates the storage class for the target environment.
   bool IsValidStorageClass(SpvStorageClass storage_class) const;
 
+  // Takes a Vulkan Valid Usage ID (VUID) and return the string only if
+  // targeting Vulkan
+  std::string VkErrorID(std::string id) const;
```

Comment:
This deserves a longer comment about how to find VUIDs in various places (e.g. html spec or from Vulkan-Docs).

Reply:
fair, I wanted to put a "here is the long detailed explanation" somewhere, but wasn't sure where, if you think this declaration is the spot to add it, happy to append this